service/frontend/thriftHandler_test.go

```diff
@@ -182,7 +183,7 @@ func TestThriftHandler(t *testing.T) {
 	t.Run("ResetWorkflowExecution", func(t *testing.T) {
 		h.EXPECT().ResetWorkflowExecution(ctx, &types.ResetWorkflowExecutionRequest{}).Return(&types.ResetWorkflowExecutionResponse{}, internalErr).Times(1)
 		resp, err := th.ResetWorkflowExecution(ctx, &shared.ResetWorkflowExecutionRequest{})
-		assert.Equal(t, shared.ResetWorkflowExecutionResponse{}, *resp)
+		assert.Equal(t, shared.ResetWorkflowExecutionResponse{RunId: common.StringPtr("")}, *resp)
 		assert.Equal(t, expectedErr, err)
 	})
```

Comment:
pretty much all of these tests don't reflect what happens on error - the implementations I can see return `nil, err`, not pointers to zero-valued responses.  that seems to be the source of the now-required runid -> `&""` in the response.

are these tests doing anything useful, aside from double-checking that types compile?  or should we just delete them?

Reply:
Yes, these needs improvement. I was planning to do something about them. For now they just check that thrift handler forwards the call to the underlying handler.